### DISCUSSION TITLE
ERROR (MainThread) [custom_components.frigate.api] Error fetching information from http://ccab4aaf-frigate:5000/api/stats:

### DISCUSSION THREAD

**Discussion by witzker** [blakeblackshear/frigate#15581] [Ask A Question]
Hi, I'm totally new with HA and frigate.

I noticed the following error in HA assistant log:

`ERROR (MainThread) [custom_components.frigate.api] Error fetching information from http://ccab4aaf-frigate:5000/api/stats: Cannot connect to host ccab4aaf-frigate:5000 ssl:default [DNS server returned answer with no data]`

Pls tell a newbie how to fix
Manx THX

**Comment by NickM-27** [comment]
have you installed frigate itself?

**Comment by witzker** [comment]
HA on bare metal frigate as add-on

**Reply by NickM-27** [reply]
which addon specifically?
